metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvesterImpl.java

```diff
@@ -216,153 +193,5 @@ public void forEachFiltered(ReportingIteration<Path> action, Predicate<Path> fil
     public String getExtractedDirectory() {
       return super.getExtractedDirectory();
     }
-
-    @Override
-    public void deleteIteratorContent() {
```

Comment:
I see this is fulfilled by the `AutoClosable` used by `AbstractHttpHarvestIterator`